tools/github_actions_update_builder.sh

```diff
@@ -2,8 +2,8 @@
 # Mapping of /output should be set in the docker run options for the
 # run-updater step of the generate job in .github/workflows/Updater.yml
 cd /output/ || exit 1
+crew update && yes | crew upgrade
```

Comment:
Would `crew upgrade -f` work too?

Reply:
Realized that `crew upgrade` doesn't work at all here because if there are unbuilt updated packages here those packages will not have binaries yet for installation.